test/border.jl

```diff
@@ -270,6 +270,13 @@ using Test
         a0 = reshape([1])  # 0-dimensional
         @test ImageFiltering.accumulate_padding((), a0) == ()
         @test ImageFiltering.accumulate_padding((0:1, -1:1), a0) == (0:1, -1:1)
+        # ImageFiltering #137
+        b_front = BorderArray(zeros(10), Fill(1.0, (3,), (0,)))
+        b_both = BorderArray(b_front, Fill(1.0, (0,), (5,)))
+        b_sim = similar(b_both)
+        @test axes(b_sim) == axes(b_both)
```

Comment:
Can you maybe also add a few very explicit `axes` tests? Like
`@test axes(b) === (IdentityUnitRange(...)),)`?

Reply:
Or rather than testing the type, maybe best would be to test with `==`, which should test both the values of the axes and the axes-of-the-axes.

Reply:
I think I'm already testing with `==` rather than testing the type?  So I'm not sure what you mean here.

Reply:
Don't just test that they are the same, test the actual value produced. E.g., if you expect `f(x)` to return 1, testing `f(x) == f(y)` will not catch certain classes of bugs.

Reply:
Ah got it, thanks!

Reply:
> Can you maybe also add a few very explicit `axes` tests? Like
> `@test axes(b) === (IdentityUnitRange(...)),)`?

I added that test, just let me know if there are others you'd like added